src/modules/ekf2/ekf2_main.cpp

```diff
@@ -269,6 +269,8 @@ class Ekf2 : public control::SuperBlock
 	_rng_aid;              // enables use of a range finder even if primary height source is not range finder (EKF2_HGT_MODE != 2)
 	control::BlockParamExtFloat _rng_aid_hor_vel_max; 	// maximum allowed horizontal velocity for range aid
 	control::BlockParamExtFloat _rng_aid_height_max; 	// maximum allowed absolute altitude (AGL) for range aid
+	control::BlockParamExtFloat
+	_rng_aid_innov_gate; // gate size used for innovation consistency checks for range aid fusion
```

Comment:
I will go through the estimator code and submit a separate PR' for this and the ecl library to make the Doxygen compatibility changes.